src/PaletteProvider.js

```diff
@@ -54,42 +50,28 @@ type Props = {
   /**
    * Render LoaderComponent when the palette is being created
    */
-  LoaderComponent?:
-    | React$Component<*, *, *>
-    | ((...args: *) => React$Element<*>),
+  LoaderComponent?: ReactClass<*> | ((...args: *) => React.Element<*>),
```

Comment:
Let's use `Array<*>` or `Array<any>` better for the rest parameter.

From the docs: `If you add a type annotation to a rest parameter, it must always explicitly be an Array type.`

https://flow.org/en/docs/types/functions/#toc-rest-parameters